### PR TITLE
[stable/node-problem-detector] Mount /var/log as readOnly

### PR DESCRIPTION
**Pull request by shulima**
## Description

The chart mounting node's `/var/log` directory with write permissions got flagged in our internal review as a security vulnerability. Local testing did not reveal any issues when switching the mount to readOnly mode.

## Checklist

- [X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)
- [X] I have read the [contribution instructions](https://github.com/deliveryhero/helm-charts#contributing), bumped chart version and regenerated the docs
- [ ] Github actions are passing

### CODE REVIEW

**Comment by max-rocket-internet** [comment]
@shulima thanks for the PR, it looks good, but you need to regenerate the readme.

**Comment by shulima** [comment]
@max-rocket-internet my laziness exposed. Oops. Regenerated.

File: stable/node-problem-detector/README.md [hunk 1]
```diff
@@ -1,42 +1,16 @@
 # node-problem-detector
 
-![Version: 2.0.3](https://img.shields.io/badge/Version-2.0.3-informational?style=flat-square) ![AppVersion: v0.8.9](https://img.shields.io/badge/AppVersion-v0.8.9-informational?style=flat-square)
+![Version: 2.0.4](https://img.shields.io/badge/Version-2.0.4-informational?style=flat-square) ![AppVersion: v0.8.9](https://img.shields.io/badge/AppVersion-v0.8.9-informational?style=flat-square)
 
 This chart installs a [node-problem-detector](https://github.com/kubernetes/node-problem-detector) daemonset. This tool aims to make various node problems visible to the upstream layers in cluster management stack. It is a daemon which runs on each node, detects node problems and reports them to apiserver.
 
 **Homepage:** <https://github.com/kubernetes/node-problem-detector>
 
-## How to install this chart
```
Comment (max-rocket-internet) [inline stable/node-problem-detector/README.md:9]: Why is all this being removed?

Comment (shulima) [reply stable/node-problem-detector/README.md:9]: No idea; I generated the README according to the contribution instructions, I assumed that the behavior was intended.

Comment (shulima) [reply stable/node-problem-detector/README.md]: I see now -- the location from which helm-docs is run from matters. The REAME looks more sane now.

**Review by max-rocket-internet — APPROVED** [review]

File: stable/node-problem-detector/README.md (hunk 1 above)
Comment (max-rocket-internet) [reply stable/node-problem-detector/README.md]: Thanks!

**Pull request merged**